src/main/java/reactor/netty/http/server/HttpServerOperations.java

```diff
@@ -403,6 +404,13 @@ public HttpServerResponse status(HttpResponseStatus status) {
 		return withWebsocketSupport(uri(), protocols, maxFramePayloadLength, proxyPing, websocketHandler);
 	}
 
+	@Override
+	public Mono<Void> sendWebsocket(
+			BiFunction<? super WebsocketInbound, ? super WebsocketOutbound, ? extends Publisher<Void>> websocketHandler,
+			WebSocketConfigurer configurer) {
+		return withWebsocketSupport(uri(), configurer.getProtocols(), configurer.getMaxFramePayloadLength(), configurer.isHandlePing(), websocketHandler);
```

Comment:
Why not pass the config object all the way to `WebsocketServerOperations`. The `withWebSocketSupport` method here is package private, so it can be changed, and the `WebsocketServerOperations` constructor is package private too.